base/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import render
+from django.utils import timezone
+from django.views.generic import TemplateView
+
+
+class BookingTemplateView(TemplateView):
+
+    template_path = ""
+
+    def get_template_names(self):
+        return [self.template_path + x
+                for x in super(BookingTemplateView, self).get_template_names()]
+
+    def localnow():
```

Comment:
Questa funzione non usa `self` o `cls`, possiamo tirarla fuori dalla classe e metterla in un file (per esempio `/base/utils.py`)